File: ws/arboleda/u2/WS24Zoo/Zoo/src/main/java/ec/edu/espe/zoo/model/Mammal.java
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
 * @author Abner Arboleda, Logic Legion, DCCO-ESPE
 */
public abstract class Mammal extends Animal {
    
    public Mammal(int id, String description, Date dateOnborn, char gender, int numberOfBones, Cage cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }
    
}

File: hw/Rodriguez/u2/hw18/Zoo/ec_edu_espe_zoo_model/mammal.py
from .animal import Animal
from datetime import date
from .cage import Cage

class Mammal(Animal):
    def __init__(self, id, description, date_onborn, gender, number_of_bones, cage):
        super().__init__(id, description, date_onborn, gender, number_of_bones, cage)
